lib/pipelines/code-pipeline.ts

```diff
@@ -172,13 +182,18 @@ export class CodePipelineBuilder implements StackBuilder {
 export class CodePipelineStack extends cdk.Stack {
 
     static readonly USAGE_ID = "qs-1s1r465k6";
+    static readonly USAGE_ID_MULTI_ACCOUNT = "qs-1s1r465f2";
```

Comment:
Where is that usage tracking ID from? I do recall we had a range, but once a new identifier is leveraged we need to communicate it to the quickstart team. I assume it is leveraged in reports.